proto/stream_plan.proto

```diff
@@ -810,6 +810,8 @@ message StreamActor {
   common.Buffer vnode_bitmap = 8;
   // The SQL definition of this materialized view. Used for debugging only.
   string mview_definition = 9;
+  // Provide the necessary context, e.g. session info like time zone, for the actor.
+  CapturedContext captured_context = 10;
```

Comment:
Yes, since in https://github.com/risingwavelabs/risingwave/pull/13647#discussion_r1404712030, we've decided to capture the context like timezone no matter it is used or not, we should always pass it to all the actors.